tests/foreman/cli/test_user.py

```diff
@@ -247,6 +253,36 @@ def test_positive_create_user_6(self):
             self.fail(err)
         self.__assert_exists(args)
 
+    def test_create_user_with_location(self):
+        """@Test: Check if user with location can be created
+
+        @Feature: User - Positive create
+
+        @Assert: User is created and has new location assigned
+
+        """
+        try:
+            location = make_location()
+            user = make_user({'location-ids': location['id']})
```

Comment:
According to #1714 the option that should be used here is `--default-location-id`